cmd/alertmanager/main.go

```diff
@@ -411,7 +411,9 @@ func main() {
 		for {
 			select {
 			case <-hup:
-				reload()
+				if err := reload(); err != nil {
+					// logged in reload()
```

Comment:
Having an empty `if` block is currently viewed as an exception by our linter

Reply:
I have opened up an issue upstream (errcheck) to ignore local functions, but so far no reply: https://github.com/kisielk/errcheck/issues/151

If nothing happens any time soon, I will pull out the logging code from `reload` as you suggested :+1:.

Reply:
@mxinden `_ = reload` should be your friend (don't forget to add a comment why you are ignoring the returned error.